File: next/content/problems/asm-working-hours/solutions/typescript/starter.ts
export function solution(queries: string[][]): string[] {
  const workers = new Map<string, unknown>();
  const result: string[] = [];

  for (const query of queries) {
    switch (query[0]) {
      case "ADD_WORKER":
        result.push("");
        break;
      case "REGISTER":
        result.push("");
        break;
      case "GET":
        result.push("");
        break;
      case "TOP_N_WORKERS":
        result.push("");
        break;
      case "PROMOTE":
        result.push("");
        break;
      case "CALC_SALARY":
        result.push("");
        break;
      case "SET_DOUBLE_PAY":
        result.push("");
        break;
      default:
        result.push("");
    }
  }

  return result;
}

File: next/content/problems/asm-working-hours/solutions/python/starter.py
def solution(queries):
    workers = {}
    result = []

    for query in queries:
        op = query[0]
        if op == "ADD_WORKER":
            # Add worker state here.
            result.append("")
        elif op == "REGISTER":
            # Toggle whether the worker is currently inside the office.
            result.append("")
        elif op == "GET":
            # Return completed working time.
            result.append("")
        elif op == "TOP_N_WORKERS":
            result.append("")
        elif op == "PROMOTE":
            result.append("")
        elif op == "CALC_SALARY":
            result.append("")
        elif op == "SET_DOUBLE_PAY":
            result.append("")
        else:
            result.append("")

    return result
